scripts/bitbucket.py

```diff
@@ -104,7 +104,16 @@ def post_result(url, user, password, verify, data, debug):
             err("SSL warnings disabled\n")
 
     # Construct the URL and JSON objects
-    post_url = url + commit_sha
+    if driver == 'Bitbucket Server':
+        post_url = j['source']['bitbucket_url'] + 'rest/build-status/1.0/commits/' + commit_sha
+    elif driver == 'Bitbucket Cloud':
+        owner = j['source']['owner']
+        repository_name = j['source']['repository_name']
```

Comment:
The `repository_name` doesn't appear to be in the README. You should just use the param `repo` passed here instead of setting up what is essentially a duplicated variable.